src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -368,11 +368,15 @@ other Axom developers to use during development, in Axom GitLab CI testing, etc.
 
    Run the corresponding command for the system you are on::
 
-     # blueos
+     # blueos_3_ppc64le_ib_p9 (default cmake is 3.14, need >=3.21 for using-with-cmake example)
+     $ module load cmake/3.29.2
```

Comment:
👍 -- thanks for adding this! My test runs for #1541 failed during the using-with-cmake examples b/c I forgot to load a newer cmake module.